app/Filker.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Filker extends Model
```

Comment:
Improve readibility: Filker should be Fylker.

Reply:
ok